src/canisters/frontend/ic-asset/src/batch_upload/operations.rs

```diff
@@ -167,3 +175,54 @@ pub(crate) fn set_encodings(
         }
     }
 }
+
+pub(crate) fn update_properties(
+    operations: &mut Vec<BatchOperationKind>,
+    project_assets: &HashMap<String, ProjectAsset>,
+    canister_asset_properties: &HashMap<String, AssetProperties>,
+) {
+    for (key, project_asset) in project_assets {
+        let project_asset_properties = project_asset.asset_descriptor.config.clone();
+        let canister_asset_properties = canister_asset_properties.get(key);
+        // skip if the asset is not already in the canister, because
+        // proporties gonna be created during create_new_assets call
+        if canister_asset_properties.is_none() {
+            continue;
+        }
+        let canister_asset_properties = canister_asset_properties.unwrap();
```

Comment:
Isn't this all the same, but without the is_none/unwrap?
```suggestion
        // skip if the asset is not already in the canister, because
        // properties gonna be created during create_new_assets call
        let Some(canister_asset_properties) = canister_asset_properties.get(key) {
            ....
        }
```